code/modules/research/xenoarchaeology/finds/finds.dm

```diff
@@ -386,11 +386,11 @@
 				new_item.icon = 'icons/obj/xenoarchaeology.dmi'
 				new_item.icon_state = "egun[rand(1,6)]"
 				new_item.item_state = new_item.icon_state
-				new_item.desc = "This is an antique energy weapon, you're not sure if it will fire or not."
+				additional_desc = "Looks like an antique energy weapon, you're not sure if it will fire or not."
 				new_item.inhand_states = list("left_hand" = 'icons/mob/in-hand/left/xenoarch.dmi', "right_hand" = 'icons/mob/in-hand/right/xenoarch.dmi')
 				if(prob(10)) // 10% chance to be a smart gun
 					new_item.can_take_pai = TRUE
-					new_item.desc += "There seems to be some sort of slot in the handle."
+					additional_desc += "There seems to be some sort of slot in the handle."
```

Comment:
Looks like an antique energy weapon, you're not sure if it will fire or not.There seems to be some sort of slot in the handle.

Notice lack of space.